src/starbase/integration.ts

```diff
@@ -17,19 +22,49 @@ async function writeIntegrationDataToNeo4j(
   );
 }
 
+async function writeIntegrationDataToJupiterOne(
+  integrationInstanceId: string,
+  integrationDirectory: string,
+  storageConfig: JupiterOneStorage,
+) {
+  const isDev = storageConfig.config.apiBaseUrl?.match(/dev/) || false;
```

Comment:
I can't recall for certain whether or not we have different API base URLs for other J1 environments like EU, but if we do I would wonder if we'd want to remove this check and always provide the `apiBaseUrl` value to the sync command so that it can be used to target those environments as well.

Reply:
agree. A passthru here is preferred.